Apps/AdminWebClient/src/ClientApp/src/views/Support.vue

```diff
@@ -121,8 +120,11 @@ export default class SupportView extends Vue {
         this.supportService = container.get(SERVICE_IDENTIFIER.SupportService);
     }
 
-    private formatDate(date: string): string {
-        return moment(date).format("l LT");
+    private formatDate(date: Date): string {
```

Comment:
We need to move away from using Date since its capabilities are limited when compared to moment or luxon.